cordformation/src/main/kotlin/net/corda/plugins/Cordformation.kt

```diff
@@ -72,6 +72,10 @@ class Cordformation : Plugin<Project> {
         val jolokia = project.dependencies.add(CORDFORMATION_TYPE, "org.jolokia:jolokia-jvm:$jolokiaVersion:agent")
         // The Jolokia agent is a fat jar really, so we don't want its transitive dependencies.
         (jolokia as ModuleDependency).isTransitive = false
+
+        val postgresDriverVersion = project.findRootProperty("postgres_driver_version") ?: "42.2.10"
+        val postgresDriver = project.dependencies.add("cordapp","org.postgresql:postgresql:$postgresDriverVersion")
```

Comment:
Yikes! Why is Cordformation adding extra dependencies to the Gradle project? I wouldn't expect that!

(Yes, I see Jolokia does this too. _**UGH!**_  OTOH, at least no-one _sees_ Jolokia and CorDapp developers hopefully don't know it even exists...)

This is a Gradle plugin, so think about what your `build.gradle` script is going to look like as you try to use this new feature. And _please_ be careful to keep backwards compatibility with older versions of the plugins so that people can upgrade their plugins without breaking their projects.

Reply:
I removed this code from Cordformation